aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/TagHelper.java

```diff
@@ -67,9 +67,9 @@
                     .stream()
                     .collect(Collectors.mapping(entry ->
                                     Tag.builder()
-                                       .key(entry.getKey())
-                                       .value(entry.getValue())
-                                       .build(),
+                                            .key(entry.getKey())
```

Comment:
this file is not changed

Reply:
yes this file has no changes, but i have formate the full file so it was showing this changes as alignment problem. 
No code changes are made on this file